src/component/fileBrowser.jsx

```diff
@@ -142,7 +144,7 @@ const LocalFileBrowser = ({ superState, dispatcher }) => {
                             });
                             window.localStorage.setItem('fileList', JSON.stringify(fileState));
                         }}
-                        directory
+                        directory="true"
```

Comment:
Why is this?

Reply:
This is to remove browser warnings ,
I have seen your comment in that issue and now I will remove this thing

And please help me to merge this "Save" issue as i think It is working fine now as required
If you have any requirements please tell me so that I full fill those